extensions/ql-vscode/src/extension.ts

```diff
@@ -1569,6 +1575,49 @@ async function activateWithInstalledDistribution(
   };
 }
 
+function addUnhandledRejectionListener() {
+  const handler = (error: unknown) => {
+    // This listener will be triggered for errors from other extensions as
+    // well as errors from this extension. We don't want to flood the user
+    // with popups about errors from other extensions, and we don't want to
+    // report them in our telemetry.
+    //
+    // The stack trace gets redacted before being sent as telemetry, but at
+    // this point in the code we have the full unredacted information.
+    const isFromThisExtension =
+      extension && getErrorStack(error).includes(extension.extensionPath);
+
```

Comment:
Nice...so this will handle the local development case as well as bundled extensions?

Reply:
It does still work for local development, because the extension path still matches the location of the installed extension. In my case it's `/Users/robertbrignull/github/vscode-codeql/extensions/ql-vscode`.